bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/Parameter.java

```diff
@@ -26,9 +26,11 @@
 public class Parameter {
     public final String name;
     public final boolean isDefault;
+    public Type type;
```

Comment:
Shouldn't these be private variables and provide getters?

@warunalakshitha @HindujaB ?

Reply:
I'll address this when fixing https://github.com/ballerina-platform/ballerina-lang/issues/31744

Reply:
Great. Thank you.